### PR TITLE
Update file_permissions.rst

### PR DESCRIPTION
**Pull request by beyerz**
Added extra explanation for setfacl to clarify the purpose of each command

### CODE REVIEW

File: setup/file_permissions.rst [hunk 1]
```diff
@@ -54,7 +54,9 @@ following script to determine your web server user and grant the needed permissi
     $ sudo setfacl -dR -m u:"$HTTPDUSER":rwX -m u:`whoami`:rwX var
 
 .. note::
-
+    The first setfacl deals with setting permissions on the currently exisitng files and folders,
```
Comment (venyii) [inline setup/file_permissions.rst]: `exisitng` -> `existing`

**Review by javiereguiluz — APPROVED** [review]
👍

**Comment by xabbuh** [comment]
👍

**Comment by xabbuh** [comment]
Thank you @beyerz.

**Pull request closed without merging**